app/components/personcircle.js

```diff
@@ -0,0 +1,23 @@
+import { Col } from "react-bootstrap";
+import styles from "../styles/Personacircle.module.css";
+import {getStrapiURL } from '../lib/api';
+export default function Personcircle({persons}) {
+  return (
+    <>
+      <Col
+        className={`${styles.personas} d-flex flex-wrap justify-content-center`}
+      >
+        {persons.map((person) => {
+          return (
+            <span className={`${styles.persona}`}>
+              <div className={styles.svg}> 
+                <img src={getStrapiURL("")+person.avatar?.url} />
```

Comment:
Why are you not passing the path as an argument? 

Plus, since we're here, can you also make sure getStrapiUrl actually sanitizes the root strapi url? Currently the || part doesn't contain any trailing slash, but the environment variable might just do.

Reply:
Hmm, I interpret wrong, but now I added the icon name inside Strapi, and dynamically fetch it from react-icon.